pkgs/tools/misc/bat-extras/default.nix

```diff
@@ -48,6 +49,7 @@ let
 
     # Run the library tests as they don't have external dependencies
     doCheck = true;
+    checkInputs = [ getconf ];
```

Comment:
This should be behind lib.optionals.

Reply:
You mean that we just set it on Darwin? I though to keep it always as it is definitively required on both, however somehow included in linux `stdenv` or somewhat else ...

Reply:
I think it is included in glibc for linux. Darwin uses clang so it is not included there. Maybe we want to build it with gccStdenv instead. If not please put it behind stdenv.isDarwin just to be sure.